packages/bridge-ui/src/pages/home/Home.svelte

```diff
@@ -1,17 +1,21 @@
 <script lang="ts">
   import { _ } from 'svelte-i18n';
-  import { link, location } from 'svelte-spa-router';
+  import { location } from 'svelte-spa-router';
   import { transactions } from '../../store/transactions';
   import BridgeForm from '../../components/form/BridgeForm.svelte';
   import TaikoBanner from '../../components/TaikoBanner.svelte';
   import Transactions from '../../components/Transactions.svelte';
+  import { Tabs, TabList, Tab, TabPanel } from '../../components/Tabs';
 
   let bridgeWidth: number;
   let bridgeHeight: number;
 
   $: activeTab = $location.replace('/', '').startsWith('transactions')
```

Comment:
this seems brittle, if we change the route name, or if we add another API route like `/transactionsForChain` and only show one chain's transactions, or something.

Reply:
Agree. Let me change that